docs/modules/operation/pages/deep-dive/alarms/alarm-example.adoc

```diff
@@ -0,0 +1,39 @@
+
+= Single alarm tracking problem states
```

Comment:
This is a screenshot-heavy page.  It should evaluated to determine if it should be kept and updated or converted to text descriptions.  This page probably could be combined with alarm-handling.

```suggestion
= Alarm Lifecycle
```